atlp-rwanda/atlp-devpulse-fn
# #217 Update profile

### PR Description

This PR allows users to view and update their profile

### Description of tasks that were expected to be completed

As an applicant I want to be able to update my profile information from my settings and also view my information if I navigate to profile.

### How has this been tested?
- Clone repo
- Checkout branch `update-profile`
- Run `npm install`, then `npm run dev` to check if the app is running successfully
- Click on `settings` on the sidebar and click `update profile` to change profile information
- To view profile, click on the profile button on the header and click profile to navigate to the profile page

### Credentials
username: `admin@example.com`
password: `password123`

username: `jean@example.com`
password: `password123`

### Track PR (issue number & link)
Issue: [#217](https://github.com/atlp-rwanda/atlp-devpulse-fn/issues/217)

### Screenshots
![Screenshot 2024-11-22 001303](https://github.com/user-attachments/assets/b3a0c914-7e70-40a3-a907-a28d14bd7c91)

![Screenshot 2024-11-22 001324](https://github.com/user-attachments/assets/7105a08e-61a9-456c-9a55-c050fb0c6272)
